static/lib/composer.js

```diff
@@ -23,14 +23,15 @@ define('composer', [
 	};
 
 	$(window).off('resize', onWindowResize).on('resize', onWindowResize);
+	onWindowResize();
 
 	$(window).on('action:composer.topics.post', function(ev, data) {
 		localStorage.removeItem('category:' + data.data.cid + ':bookmark');
 		localStorage.removeItem('category:' + data.data.cid + ':bookmark:clicked');
 	});
 
 	$(window).on('popstate', function() {
-		var env = utils.findBootstrapEnvironment();
```

Comment:
If I recall correctly, I call `utils.findBootstrapEnvironment` here instead of caching because the browser window could have resized.

Esp. ipad, when portrait to landscape, could trigger mobile->desktop behaviour...